Configuration/AlCa/python/autoCond.py

```diff
@@ -36,11 +36,9 @@
     # GlobalTag for Run2 HLT for HI: it points to the online GT
     'run2_hlt_hi'       :   '81X_dataRun2_HLTHI_frozen_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2017
-    'phase1_2017_design' :  '81X_upgrade2017_design_v12',
+    'phase1_2017_design' :  '81X_upgrade2017_HCALdev_v3',
```

Comment:
@makortel, depending on your patience, you might decide to update this PR to update this key to contain what is supposed to contain, i.e. ideal conditions, but 2017 version of Hcal calibrations to:

`'phase1_2017_design' :  '81X_upgrade2017_design_v13'`

diff with respect to previous:

https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/81X_upgrade2017_design_v12/81X_upgrade2017_design_v13